__tests__/actionCreators/resources.test.js

```diff
@@ -206,20 +212,23 @@ describe('existingResource', () => {
     it('dispatches no actions', async () => {
       resourceTemplateThunks.fetchResourceTemplate = jest.fn().mockResolvedValue(undefined)
 
-      await store.dispatch(existingResource(resource, uri))
+      await store.dispatch(existingResource(resource, null, uri))
       const actions = store.getActions()
       expect(actions.length).toEqual(0)
     })
   })
+
   describe('when stubResource returns a result', () => {
     it('dispatches actions', async () => {
       const resourceTemplateResponse = await getFixtureResourceTemplate(resourceTemplateId)
       const resourceTemplate = resourceTemplateResponse.response.body
       resourceTemplateThunks.fetchResourceTemplate = jest.fn().mockResolvedValue(resourceTemplate)
 
-      await store.dispatch(existingResource(resource, uri))
+      const unusedRDF = '<> <http://id.loc.gov/ontologies/bibframe/mainTitle> "foo"@en .'
+
+      await store.dispatch(existingResource(resource, unusedRDF, uri))
       const actions = store.getActions()
-      expect(actions.length).toEqual(2)
+      expect(actions.length).toEqual(3)
```

Comment:
Would it be clearer to write `expect(actions).toEqual([{type: 'SET_RESOURCE'}, ...]) `

Reply:
line 231 is now unnecessary since you've asserted the same on 238.

Reply:
I wanted to be extra sure.

(Removed)